cdc/entry/schema/snapshot.go

```diff
@@ -194,9 +200,36 @@ func NewEmptySnapshot(forceReplicate bool) *Snapshot {
 		forceReplicate:   forceReplicate,
 		currentTs:        0,
 	}
+
 	return &Snapshot{inner: inner, rwlock: new(sync.RWMutex)}
 }
 
+// InitConcurrentDDLTables imitate the creating table logic for concurrent DDL.
+// since v6.2.0, tables of concurrent DDL will be directly written to meta
+// without written to history DDL jobs.
+func (s *Snapshot) InitConcurrentDDLTables() {
+	mysqlDBInfo := &timodel.DBInfo{
+		ID:      1,
+		Name:    timodel.NewCIStr(mysql.SystemDB),
+		Charset: mysql.UTF8MB4Charset,
+		Collate: mysql.UTF8MB4DefaultCollation,
+		State:   timodel.StatePublic,
+	}
+	mockTS := uint64(1)
```

Comment:
Please comment on the purpose and the value of `mockTS`. Also, I think `dummyTs` can be a better name? Also, can you make it a const?

Reply:
https://github.com/pingcap/tiflow/pull/6485/commits/700b3a98505876cc05f56d332277ff011880945a